src/components/TodoForm.vue

```diff
@@ -2,28 +2,48 @@
   <div class="ToDoList">
     <Header @todoAdded="addTodo" />
     <ClearAll @clearAll="clearAll" />
-    <ul>
-      <li v-for="(toDo, index) in todos" :key="index">
-        {{ toDo }}
-        <RemoveToDo @todoRemoved="removeTodo(index)" />
-      </li>
+    <input
+      v-if="todos.length"
+      maxlength="50"
+      type="text"
+      v-model="newTodo"
```

Comment:
Respect the order of the attributes according to style guide the `v-model` should be in the very top right after `v-if` it is important to be easily identified the variable you changing